CMake/BuildConfigurations/CaPTkAppAll.cmake

```diff
@@ -12,6 +12,7 @@ set(MITK_USE_Python3 ON CACHE BOOL "" FORCE)
 
 # Enable/disable non-superbuild apps
 set(MITK_BUILD_APP_CaPTkApp ON CACHE BOOL "Build the MITK CaPTkApp" FORCE)
+set(MITK_BUILD_APP_InciSe OFF CACHE BOOL "Build the MITK InciSe executable" FORCE)
```

Comment:
This is needed to build the superbuild from scratch with the CaPTkAll config. It is not really related with the new incise changes